modules/serverless-create-trigger-odc.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-create-trigger-odc_{context}"]
+= Creating a trigger using the Developer perspective
+
+After you have created a broker, you can create a trigger in the web console *Developer* perspective.
+
+.Prerequisites
+To create triggers using the *Developer* perspective ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created a broker and a Knative service or other event sink to connect to the trigger.
+
+.Procedure
+
+. In the *Developer* perspective, navigate to the *Topology* page.
+. Hover over the broker that you want to create a trigger for, and drag the arrow. The *Add Trigger* option is displayed.
++
+image::add-trigger-odc.png[Create a trigger for the broker]
```

Comment:
Just saying that this screenshot seems unnecessarily huge.

Reply:
I'd need to spin up a cluster to get a new image to change this, since I think it's just that the image needs to be zoomed out maybe. I'll follow this up another time if that's OK?

Reply:
@abrennan89 Actually I just happened upon an update from @adellape on how you can adjust the size of the image in AsciiDoc.

His example was:
```
image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40]
```

Not sure the size that would make sense for your image, but should be easy enough to play around with. I agree though that it's fine to follow up on later if you don't have the time now though